DOUBLE_CLICK_AND_EXECUTE_ME.py

```diff
@@ -15,65 +15,48 @@
 root.title("Dell'Osso Line Program")
 
 # Set window size
-#root.geometry("1920x1080")
+# root.geometry("800x600")
 root.attributes('-fullscreen', True)
 
 # Sets the background color
 root.configure(bg="black")
 
 # NEXT WINDOW heading
-label_heading = Label(root, text="PROCEED TO WINDOW", font=("Chalkduster", 100),
-                      fg="red", bg="black") 
-
-# Timer for NEXT WINDOW
-label_heading.timer = ' '
-
-# Pack label_heading
-label_heading.pack(pady=(30, 30))
-
-# Create a Frame to contain the numbers 1 through 6
-number_frame = Frame(root, bg="black")
-
-# Number 1 through 6
-nmbr_1 = Label(number_frame, text =" 1 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_1.pack(padx=15, side=LEFT)
-nmbr_2 = Label(number_frame, text =" 2 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_2.pack(padx=15, side=LEFT)
-nmbr_3 = Label(number_frame, text =" 3 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_3.pack(padx=15, side=LEFT)
-nmbr_4 = Label(number_frame, text =" 4 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_4.pack(padx=15, side=LEFT)
-nmbr_5 = Label(number_frame, text =" 5 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_5.pack(padx=15, side=LEFT)
-nmbr_6 = Label(number_frame, text =" 6 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_6.pack(padx=15, side=LEFT)
+row_heading_labels = []
+row_heading_labels.append(Label(root, text="PROCEED TO WINDOW", font=("Chalkduster", 100), fg="red", bg="black"))
+row_heading_labels[0].pack(pady=(30, 30))
+
+# Create number frames to hold the rows of numbers
+number_frames = []
+number_frames.append(Frame(root, bg="black"))
+number_frames.append(Frame(root, bg="black"))
+
+# Number labels
+number_labels = {}
+for i in range(1, 12):
+    frame = number_frames[0 if i < 7 else 1]
+    number_labels[i] = dict()
```

Comment:
I'm not sure I understand lines 38-39. It looks like you create a dictionary but I don't understand how you're making a key/value pairing. I googled the `%s` operator and I have more questions that answers on that lol.

Reply:
So I created a dictionary thinking at the point that I need a timer for each number (which to get the exact functionality you had before we would) but since I was debounced the input the canceling of the previous timer was not important so I ended up not using individual timers.  But the concept was a dictionary with a "label" key and a "timer" key.

As for the %s it is a way to include a variable in a string.   %s = string, there is also %i for integer and %f for float and others.  It is kinda a deprecated way since there is newer ways (format, f strings), but when I used an f string it gave me an error in the context of setting a parameter (not sure why).  So " %s " % i = " 1 ", then " 2 ", etc. for each number in the range.